src/delta_utils/check_submission.py

```diff
@@ -82,9 +83,7 @@ def check_submission(
     try:
         choose_move = getattr(mod, "choose_move")
     except AttributeError as e:
-        raise Exception(
-            f"No function 'choose_move()' found in file {file_name}.py"
-        ) from e
+        raise Exception(f"No function 'choose_move()' found in file {file_name}.py") from e
```

Comment:
I know you didn't change this, but sourcery wants us to raise a specfic exception, does it make sense to raise an AttributeError given that's what it is?

Reply:
Sure